test/plugins/footnote/insert.js

```diff
@@ -0,0 +1,21 @@
+/** @jsx h */
+
+export default editor => {
+    editor.insertInline({
+        type: 'footnote',
+        data: { collapse: false },
+        nodes: [{ object: 'text', text: ' ' }],
+    }).moveBackward()
```

Comment:
When a user inserts a footnote in the editor it should be empty, and test cases should reflect that.

Reply:
If it's empty then it will be removed and there is no way to insert cursor in it.
This is way it's added with space inside. Do you see another option to handle this case?

Reply:
Unfortunately no.